### PR TITLE
1186165980 Change database lib to databases

### PR DESCRIPTION
**Pull request by zsinx6**

### CODE REVIEW

**Comment by jamesbeedy** [comment]
seeing 

```
CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
    from cryptography.utils import int_from_bytes, int_to_bytes

-- Docs: https://docs.pytest.org/en/latest/warnings.html
``` 

in the `make test` output

Can we rid that warning pls?

File: jobbergateapi2/config.py [hunk 1]
```diff
@@ -3,10 +3,11 @@
 """
 from pydantic import BaseSettings, Field
 
+_DB_RX = r"^(sqlite|postgres)://.+$"
```
Comment (jamesbeedy) [inline jobbergateapi2/config.py:6]: 👍

**Review by jamesbeedy — APPROVED** [review]
Test pass on my local, runtime function seems to be 👍 . LGTM

**Comment by jamesbeedy** [comment]
100% - thank you

**Comment by jamesbeedy** [comment]
1186149976

File: jobbergateapi2/config.py [hunk 2]
```diff
@@ -3,10 +3,11 @@
...
+
 
 class Settings(BaseSettings):
-    DATABASE_URL: str
-    TEST_DATABASE_URL: str
+    DATABASE_URL: str = Field("sqlite:///./sqlite.db?check_same_thread=true", regex=_DB_RX)
     TEST_ENV: bool = Field(False)
     SECRET_KEY: str
```
Comment (matheushent) [inline jobbergateapi2/config.py:12]: Possibly usage here is [SecretStr](https://pydantic-docs.helpmanual.io/usage/types/#secret-types)

**Review by matheushent — APPROVED** [review]

**Pull request merged**